bundesliga_app/tests.py

```diff
@@ -597,24 +595,70 @@ def test_selected_events(self,
                 self.response.context_data['already_selected_id'],
             )
 
-    # def test_post_events(self,
-    #                      mock_get_events_user_eb_api,
-    #                      mock_get_user_eb_api,
-    #                      ):
-    #     event1 = mock_get_events_user_eb_api.return_value[0]
-    #     self.response = self.client.post(
-    #         '/select_events/',
-    #         {
-    #             'event_' + event1['id']: 'on',
-    #         }
-    #     )
-        # import ipdb;ipdb.set_trace()
-        # self.assertEqual(self.response.status_code, 302)
-        # # self.assertContains(
-        # #     self.response,
-        # #     event1['id'],
-        # # )
-        # self.assertEqual(
-        #     len(Event.objects.filter(event_id=event1['id'])),
-        #     1,
-        # )
+    def test_post_event_new(self,
+                            mock_get_events_user_eb_api,
+                            mock_get_user_eb_api,
+                            ):
+        event_mock_api_eb = mock_get_events_user_eb_api.return_value[0]
+        self.response = self.client.post(
+            path='/select_events/',
+            data=urlencode({
+                'event_' + event_mock_api_eb['id']: 'on',
+            }),
+            content_type='application/x-www-form-urlencoded'
+        )
+        self.assertEqual(self.response.status_code, 302)
+        self.assertEqual(
+            len(Event.objects.filter(event_id=event_mock_api_eb['id'])),
+            1,
+        )
```

Comment:
This test only selects one event. We could test if the orgniazer selects more than one event at once.